tracer/test/snapshots/Couchbase3Tests.verified.txt

```diff
@@ -0,0 +1,293 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: couchbase.query,
+    Resource: Delete,
+    Service: Samples.Couchbase3-couchbase,
+    Type: db,
+    Tags: {
+      component: Couchbase,
+      couchbase.operation.code: Delete,
```

Comment:
I don't recall why I decided not to have better resource names. But those resource names are just sad. I wonder if we could just concatenate `couchbase.operation.code` and `couchbase.operation.key`.

Reply:
Yeah, they're a bit sad, but changing them is a breaking change, so I don't think we can really do that at this point 🙁

Reply:
I'll create a ticket for the next major